client/scss/components/_dropdown.scss

```diff
@@ -74,6 +74,7 @@
   // Override any right alignment that might've been set by a parent element
   // (such as the snippets header)
   text-align: start;
+  border: 2px solid transparent;
```

Comment:
This also has an undesirable effect for regular users; please move this into a forced-colors media query.